ir/instr.cpp

```diff
@@ -2729,6 +2735,13 @@ StateValue ICmp::toSMT(State &s) const {
     UNREACHABLE();
   };
 
+  fn = [this, fn](const expr &av, const expr &bv, Cond cond) {
```

Comment:
please change the function to return a StateValue instead of smt::expr. This BV concatenation is a bit of a hack.

Reply:
Done.